src/System.Private.Uri/tests/FunctionalTests/UriIpHostTest.cs

```diff
@@ -234,6 +234,31 @@ private void ParseBadIPv4Address(string badIpv4String)
 
         #region IPv6
 
+        [Theory]
+        [InlineData("fe80::e077:c9a3:eeba:b8e9", "%18")]
+        [InlineData("Fe80::e077:c9a3:eeba:b8e9", "%18")]
+        [InlineData("fE80::e077:c9a3:eeba:b8e9", "%18")]
+        [InlineData("FE80::e077:c9a3:eeba:b8e9", "%18")]
+        [InlineData("FE80::e077:c9a3:eeba:b8e9", "%eth10")]
+        [SkipOnTargetFramework(TargetFrameworkMonikers.NetFramework)]
```

Comment:
@davidsh do you consider porting on netfx?

Reply:
I will mark it as "consider".  But those things have longer lead times for review and it might not qualify due to strict app-compat guidelines.